### DISCUSSION TITLE
Parent build

### DISCUSSION THREAD

**Discussion by mabroens** [inventree/InvenTree#4909] [Q&A]
Hi all,

can anyone teel me the idea behind **parent build**?

Reason is following.
i have a sales order that will be produced in batches so currently i entered 5 build orders. Allocated components to all build orders so i now what stock i have left. but then i noticed the parent build so started wondering if this is what i was looking for and if it would make any difference. i scanned the docs but could not find any input regarding parent build.

any suggestions thoughts?

Br Martijn

**Comment by SchrodingersGat** [comment]
I guess it needs some docs. The "parent build" allows build orders to be created in a hierarchy. Say you are making 10x Assembly A, which need 250x Subassembly B. So from the build order for A you can make a new "child" build order to produce the required quantity of B. Make sense?
